src/clr-angular/data/datagrid/providers/selection.spec.ts

```diff
@@ -186,10 +186,8 @@ export default function(): void {
 
       it('does not emit selection change twice after a filter is applied', function() {
         let nbChanges = 0;
-        let currentSelection: any;
```

Comment:
I dont know why the linter wakes up all of a sudden and starts to complain on random files.

Reply:
I caught this in another situation, agree its confusing but could be a new version of the linter.